client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -183,10 +184,26 @@ public static StreamManager create(ClientConfig clientConfig) {
      * @param scopeName  The name of the scope to delete.
      * @param forceDelete To list and delete streams, key-value tables and reader groups in scope before attempting to delete scope.
      * @return True if scope is deleted, false otherwise. 
-     * @throws DeleteScopeFailedException is thrown if this method is unable to seal and delete a stream.  
+     * @throws DeleteScopeFailedException is thrown if this method is unable to seal and delete a stream.
+     *
+     * @deprecated As of Pravega release 0.10, replaced by {@link #deleteScopeRecursive(String)}.
```

Comment:
This would be deprecated in Pravega release 0.11.0+, since 0.10.0 is already out.

Reply:
updated